src/Microsoft.ML.TimeSeries/TimeSeriesStatic.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.TimeSeriesProcessing;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+using System;
+using System.Collections.Generic;
+using static Microsoft.ML.Runtime.TimeSeriesProcessing.SequentialAnomalyDetectionTransformBase<System.Single, Microsoft.ML.Runtime.TimeSeriesProcessing.IidAnomalyDetectionBase.State>;
+
+namespace Microsoft.ML.StaticPipe
+{
+    /// <summary>
+    /// IidChangePoint static API extension methods.
+    /// </summary>
+    public static class IidChangePointStaticExtensions
+    {
+        public class Prediction
```

Comment:
@ganik Is is better to have specified fields in this class, or simply one array of size 4?